### ISSUE TITLE
Add module descriptions to the game

### ISSUE THREAD

**Issue by Asmageddon** [PavelZinchenko/event-horizon-frontier-builds#350]
The function of lots of modules is fairly unclear, especially weapons with special effects, energy shields, and other modules whose function goes beyond their raw stats.

**Comment by Absolutan115** [comment]
Damage 10 × (1000%)

**Comment by xdeadstokex** [comment]
Agree, and not until recently did I learn that ships gun slots have a firerate limit of 20 shot/s

**Comment by Asmageddon** [comment]
> Agree, and not until recently did I learn that ships gun slots have a firerate limit of 20 shot/s

Wait, what? What do you mean? Per weapon? Or in total?

**Comment by xdeadstokex** [comment]
@Asmageddon for ex: look at the slime, it have 3 weapon slots but only 1 gun slot, gun slot mean where projectile will be shot out, add 2 attack satellites will mean adding 2 more gun slots.
 I heard this fact from another post here, I tested it with the slime (small plasma cannon as the main weapon), I noted that if I use -cooldown modifer, I will get less dps than the +damage modifer, bcs the -cooldown made 3 small plasma cannons have a total firerate over 20shot/s, while the +damage total firerate is under 20shot/s.
So yeah, dont use too much automatic reloader, it will be useless, also another fact, the first shot of lots of weapons dont have cooldown (useful for high damage per shot but slow reload weapons)
Also, just want to share my most beloved ship, SLiMe
![image](https://github.com/user-attachments/assets/1c74f50a-f43d-4357-9963-56dbbd32a38e)

**Comment by Asmageddon** [comment]
...so basically, if I use 10 Small Plasma Cannons, instead of 100 shots/s of 10 damage each, I only get 20, for a total of 1/5 the damage output I wanted? For real? ._.

**Comment by xdeadstokex** [comment]
@Asmageddon AYe, mate, I also discovered this fact before I heard it, but at that time I thought it was a bug related to firerate, but no, it was not.

**Comment by Asmageddon** [comment]
@xdeadstokex How does it work with multiple separate red areas? What if you connect them with slot expansion, like possible on e.g. Embryo and Embryo MK2? What about continuous laser beams?

This seriously needs to be explained ingame... the more I play this game the more horrified I am at how many obscure, non-obvious mechanics it has.

**Comment by xdeadstokex** [comment]
@Asmageddon As I say, gun slots are where projectile will be shot out, so yes it is multiple red areas, so embryo will have 3 gun slots (3 tails).
Laser beam, hmmmmmm, I totally forgot about that cause I heavily dislike laser weapon in every games I play lol, idk how laser beam will interact with that, but it can be tested easily with 3 small plasma cannons and a small laser beam.
Indeed, me too is terrified about what I dont know about this game (like those hidden ships and weapons), so I usually only focus on building fun ships idea, that make me forget the terror.
